addons/product/views/product_attribute_views.xml

```diff
@@ -118,7 +119,7 @@
                     <field name="name" />
                     <field name="price_extra" />
                     <field name="exclude_for" widget="one2many" mode="tree">
-                        <tree editable="top">
+                        <tree editable="bottom">
```

Comment:
One little last nice to have thing, could you add a little sheet tag on this form?
We don't need the whole screen to define the exclusions anyway.
(I find it a bit ugly when coming from template > Configure Variants > Clicking on a ptav)

Reply:
Suggestion accepted by PO.